athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/serde/v24/FederationResponseSerDe.java

```diff
@@ -0,0 +1,53 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation SDK
+ * %%
+ * Copyright (C) 2019 - 2020 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connector.lambda.serde.v24;
```

Comment:
I'm wondering if we should add an additional field to ping response that lets us identify the serialization separately from the application level capabilities. For example, if the connector supports filtering then maybe we can disable filtering in Athena but that is independent of serialization. I'm not sure this is a one-way door that we can't change/add later, worse case we will have a slight confusing protocol field name. Thoughts on if we should adjust this in the GA spec?